modules/ROOT/examples/live-demos/file-picker/index.js

```diff
@@ -18,19 +18,11 @@ tinymce.init({
     input.setAttribute('type', 'file');
     input.setAttribute('accept', 'image/*');
 
-    /*
```

Comment:
I removed this as given we no longer support IE, etc... it seemed unnecessary. As for the other fixes, this is something we missed previously but the `onload` function was using `this` as such this demo no longer works when attempting to pick a local image.